samples/force-sensitive-resistor/FsrWithCapacitorSample.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+using System.Threading;
+
+namespace force_sensitive_resistor
+{
+    class FsrWithCapacitorSample
+    {
+        private GpioController _controller;
+        private int _pinNumber = 18; // set the pin number reading
+
+        public FsrWithCapacitorSample()
+        {
+            _controller = new GpioController();
+            _controller.OpenPin(_pinNumber);
+        }
+
+        private long ReadCapacitorChargingDuration()
+        {
+            int count = 0;
+            _controller.SetPinMode(_pinNumber, PinMode.Output);
+            // set pin to low
+            _controller.Write(_pinNumber, PinValue.Low);
+            // Prepare pin for input and wait until high read
+            _controller.SetPinMode(_pinNumber, PinMode.Input);
+
+            while (_controller.Read(_pinNumber) == PinValue.Low)
```

Comment:
can you connect it in such a way that it will be low when it is pressed and high when it is not pressed and ignore the delays completely?

Reply:
That actually could work just for determining on/off, but still people could use the count number for determining how much it is pressed ...

Reply:
what count? it should be 0 or 1 since you read digital pin?

Reply:
The count measuring how long it took to get 1. Yes it is digital input and can be used directly without counting or delay to get 0/1, but that count could be useful for measuring pressure intensity, really depend on usage

Reply:
why not use stopwatch for this? The `count` term is confusing since it actually usually is referred as ticks. This function should return double and the unit should be clear (second? millisecond? microsecond?)

Reply:
stopwatch is not appropriate because the thread sleeps in the calling app https://github.com/dotnet/iot/pull/333/files/33f81b8a125095dab07cf203d9d139ed2ad72d29#diff-145d59a2af55d0554553d3f750daf878R55 which causing interruption between stopwatch count and capasitor charging count which leads to invalid result.